pl_examples/basic_examples/autoencoder.py

```diff
@@ -23,7 +23,7 @@
 try:
     from torchvision.datasets.mnist import MNIST
     from torchvision import transforms
-except Exception as e:
+except ImportError:
```

Comment:
```suggestion
except ModuleNotFoundError:
```

Reply:
ok, just for the record https://airbrake.io/blog/python-exception-handling/importerror-and-modulenotfounderror